compliance_checker/tests/test_cli.py

```diff
@@ -91,7 +91,7 @@ def checker_1():
         def checker_2():
             return Namespace(_cc_spec="checker_2", _cc_spec_version="2.2")
 
-        mock_checkers = [Namespace(resolve=checker_1), Namespace(resolve=checker_2)]
+        mock_checkers = [Namespace(load=checker_1), Namespace(load=checker_2)]
```

Comment:
Maybe we could do these merges one at a time so we can investigate the impact of this one.

Reply:
CLI in general does not work right now:

```
(Pdb) ll
 68         @classmethod
 69         def _get_generator_plugins(cls):
 70             """
 71             Return a list of classes from external plugins that are used to
 72             generate checker classes
 73             """
 74  
 75             if not hasattr(cls, "suite_generators"):
 76  ->             gens = importlib_metadata.entry_points(
 77                     groups="compliance_checker.generators",
 78                 )
 79                 cls.suite_generators = [x.load() for x in gens]
 80  
 81             return cls.suite_generators
```

Something's failing in the entrypoints:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/importlib_metadata/compat/py39.py", line 31, in ep_matches
    return ep.matches(**params)
  File "/usr/local/lib/python3.9/site-packages/importlib_metadata/__init__.py", line 231, in matches
    return all(map(operator.eq, params.values(), attrs))
  File "/usr/local/lib/python3.9/site-packages/importlib_metadata/__init__.py", line 230, in <genexpr>
    attrs = (getattr(self, param) for param in params)
AttributeError: 'EntryPoint' object has no attribute 'groups'
```

Reply:
Think this should be `group`, not `groups` argument:

https://docs.python.org/3/library/importlib.metadata.html#entry-points